wherehows-web/app/styles/components/dataset-compliance/_compliance-prompts.scss

```diff
@@ -27,3 +27,25 @@
     background-color: tint(set-color(green, green5), 65%);
   }
 }
+
+/// Wraps a status notification icon for a given dataset
+.dataset-tag-container {
+  width: 20px;
+  display: inline-block;
+}
+
+/// A notification that a dataset contains a tag
+.dataset-tagged {
```

Comment:
Kinda, I made them separate components but I should rename this to `some-icon-class-name--tagged`